pkg/configtx/application.go

```diff
@@ -262,7 +262,7 @@ func newApplicationGroup(application Application) (*cb.ConfigGroup, error) {
 	applicationGroup := newConfigGroup()
 	applicationGroup.ModPolicy = AdminsPolicyKey
 
-	if err = addPolicies(applicationGroup, application.Policies, AdminsPolicyKey); err != nil {
+	if err = setPolicies(applicationGroup, application.Policies, AdminsPolicyKey); err != nil {
```

Comment:
Cleanup unrelated to the overall goal of this PR but it stood out as I was looking at the new API calls and this underlying private function's naming was inconsistent.